src/test/java/org/opentripplanner/api/common/PlannerResourceTest.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.api.common;
+
+import static graphql.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import jakarta.ws.rs.core.MultivaluedHashMap;
+import jakarta.ws.rs.core.MultivaluedMap;
+import java.util.HashMap;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner._support.time.ZoneIds;
+import org.opentripplanner.api.resource.PlannerResource;
+import org.opentripplanner.raptor.configure.RaptorConfig;
+import org.opentripplanner.routing.api.request.RequestModes;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.standalone.config.RouterConfig;
+import org.opentripplanner.standalone.configure.ConstructApplicationModule;
+import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.TransitModel;
+
+class PlannerResourceTest {
+
+  @Test
+  void defaultValues() {
+    var transitModel = new TransitModel();
+    transitModel.initTimeZone(ZoneIds.BERLIN);
+    var transitService = new DefaultTransitService(transitModel);
+    var module = new ConstructApplicationModule();
+    var context = module.providesServerContext(
```

Comment:
You shouldn't create these by hand, but let dagger take care of it

Reply:
Hmm, @t2gran gave me the tip: https://github.com/opentripplanner/OpenTripPlanner/pull/4706#discussion_r1073806590

Reply:
If I wanted to let dagger build that in a test, how would I do it?

Reply:
Ah, in that case it is ok